lib/chef/resource/service.rb

```diff
@@ -152,6 +153,14 @@ def masked(arg = nil)
         )
       end
 
+      def options(arg = nil)
+        set_or_return(
+          :options,
+          arg.respond_to?(:split) ? arg.split(/\s+/) : arg,
```

Comment:
should use `#shellsplit` i think from `shellwords` module here:

https://github.com/chef/chef/pull/5838/files